web_tool_script_3.R

```diff
@@ -395,8 +395,8 @@ if (dir.exists(exec_summary_template_path) && (peer_group %in% c("assetmanager",
       portfolio_name = portfolio_name,
       peer_group = peer_group,
       start_year = start_year,
-      scenario_source = "GECO2023",
-      scenario_selected = "1.5C",
+      scenario_source = "WEO2023",
+      scenario_selected = "NZE",
```

Comment:
```suggestion
      scenario_selected = "NZE_2050",
```

The April 24th run of data prep has this specification for scenario name:
<img width="1060" alt="Screenshot 2024-06-03 at 16 42 45" src="https://github.com/RMI-PACTA/workflow.transition.monitor/assets/8741309/f3e3c742-67fb-416e-9f3b-b140fc2f0133">

Also in line with what the config file expects:
https://github.com/RMI-PACTA/workflow.transition.monitor/blob/7591a74a826a02fa12b86bb41ef8b8ea71931355/parameter_files/ProjectParameters_PA2024CH.yml#L14-L15

Reply:
This is also how it is specified in `pacta.portfolio.report`:
https://github.com/RMI-PACTA/pacta.portfolio.report/blob/0c1837ea39e4f9acfffdacb53bd049c11048bb3c/inst/js/trajectory_alignment.js#L423-L428

Reply:
for future reference, the scenario-source pairs are included in the manifest
``` r
jsonlite::read_json("~/Downloads/2023Q4_20240527T092929Z/manifest.json")$parameters$output_stats$source_scenario_pairs
```

Reply:
or better...
``` r
manifest <- jsonlite::fromJSON("~/Downloads/2023Q4_20240527T092929Z/manifest.json")
manifest$parameters$output_stats$source_scenario_pairs
#>   scenario_source  scenario
#> 1        GECO2023      1.5C
#> 2        GECO2023   NDC-LTS
#> 3        GECO2023 Reference
#> 4         ISF2023     1.5°C
#> 5         WEO2023       APS
#> 6         WEO2023  NZE_2050
#> 7         WEO2023     STEPS
```